File: tests/testdata/codegen/simple-splat-pp/dotnet/simple-splat.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using Splat = Codeinfra.Splat;

return await Deployment.RunAsync(() => 
{
    var allKeys = Splat.GetSshKeys.Invoke();

    var main = new Splat.Server("main", new()
    {
        SshKeys = allKeys.Apply(getSshKeysResult => getSshKeysResult.SshKeys).Select(__item => __item.Name).ToList(),
    });

});

File: tests/testdata/codegen/simple-splat-pp/nodejs/simple-splat.ts
import * as codeinfra from "@codeinfra/codeinfra";
import * as splat from "@codeinfra/splat";

const allKeys = splat.getSshKeys({});
const main = new splat.Server("main", {sshKeys: allKeys.then(allKeys => allKeys.sshKeys.map(__item => __item.name))});
